src/Tribe/RSVP/Early_Access/Update_Notice.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Tribe\Tickets\RSVP\Early_Access;
+
+use Tribe__Tickets__Main;
+use Tribe__Settings;
+use Tribe__Admin__Notices;
+
+class Update_Notice {
+
+	/**
+	 * @var Early_Access
+	 */
+	private $early_access;
+
+	/**
+	 * The minimum version of Event Tickets needed to
+	 * display the update notice.
+	 *`
+	 * @since TBD
+	 */
+	private $minimum_version_to_show = 'TBD';
```

Comment:
:no_entry_sign: Missing @var tag in member variable comment

<sub>hash: cbffefc2e7433d4520aa33a0b501990b</sub>